modules/Members/pages/members.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ *  Made by Aberdeener
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.1.0
+ *
+ *  License: MIT
+ *
+ *  Member list page
+ */
+
+if (count(MemberList::getInstance()->allEnabledLists()) == 0) {
+    Redirect::to(URL::build('/'));
+}
+
+if (Input::exists()) {
+    if (Token::check()) {
+        $search = Input::get('search');
+        if (strlen($search) < 3) {
```

Comment:
Was there something which came up recently about OG Minecraft usernames having 2 characters? Maybe we could decrease this to 2 if so